tests/test_detectionlists_actions.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from py42.response import Py42Response
+from requests import Response
+
+from phantom.action_result import ActionResult
+from tests.conftest import create_fake_connector
+
+
+class TestCode42DetectionListsConnector(object):
```

Comment:
might be good to test that `handle_action` returns success / failure under the appropriate circumstances